src/WebJobs.Extensions.DurableTask/DurableTaskExtension.cs

```diff
@@ -505,9 +506,9 @@ internal string GetIntputOutputTrace(string rawInputOutputData)
             return this.httpApiHandler.CreateCheckStatusResponse(request, instanceId, attribute);
         }
 
-        // Get a response that will wait for response from the durable function for predefined period of time before
-        // pointing to our webhook handler.
-        internal async Task<HttpResponseMessage> WaitForCompletionOrCreateCheckStatusResponseAsync(
+        // Get a response that will wait for response from the durable function for predefined period of time before 
+        // pointing to our webhook handler. 
+        protected internal virtual async Task<HttpResponseMessage> WaitForCompletionOrCreateCheckStatusResponseAsync(

```

Comment:
Same comment as previous.

Reply:
I will revert these changes based on your feedback. Thank you!